src/app/app.routes.ts

```diff
@@ -29,6 +30,10 @@ export const ROUTES: Routes = [
     canActivate: [AuthGuard],
     loadChildren: '../register/register.module#RegisterModule'
   },
+  {
+    path: 'service-down',
+    component: ShutterComponent
```

Comment:
not sure if service-down and StutterComponent should be called the same...

Reply:
Changed component name